tests/api_interface/test_runbooks/test_files/decision_task.py

```diff
@@ -0,0 +1,93 @@
+"""
+Calm DSL Decision Task Example
+
+"""
+
+from calm.dsl.builtins import runbook
+from calm.dsl.builtins import CalmTask
+from calm.dsl.builtins import CalmEndpoint, ref
+from calm.dsl.builtins import read_local_file, basic_cred
+
+linux_ip = read_local_file(".tests/runbook_tests/vm_ip")
+windows_ip = read_local_file(".tests/runbook_tests/windows_vm_ip")
+CRED_USERNAME = read_local_file(".tests/runbook_tests/username")
+CRED_WINDOWS_USERNAME = read_local_file(".tests/runbook_tests/windows_username")
+CRED_PASSWORD = read_local_file(".tests/runbook_tests/password")
+
+LinuxCred = basic_cred(CRED_USERNAME, CRED_PASSWORD, name="linux_cred")
+WindowsCred = basic_cred(CRED_WINDOWS_USERNAME, CRED_PASSWORD, name="windows_cred")
+
+linux_endpoint = CalmEndpoint.Linux.ip([linux_ip], cred=LinuxCred)
+windows_endpoint = CalmEndpoint.Windows.ip([windows_ip], cred=WindowsCred)
+
+
+@runbook
+def DecisionTask(endpoints=[linux_endpoint, windows_endpoint]):
+    "Runbook Service example"
+    with CalmTask.Decision.ssh(script="exit 0", target=ref(linux_endpoint)) as val:
+
+        if val.true:
```

Comment:
There should be a straightforward way to check decision status/exitcode. As decision task is more of a `switch` than `if/else`. Please change it to `val.status = `"<SUCCESS/FAILURE>"` or/and `val.exit_code = <0/1>` so that it can be easily extended for more `cases`.